diff --git a/node_modules/ast-types/def/core.js b/node_modules/ast-types/def/core.js
index 0ed7f36..8bcda9a 100644
--- a/node_modules/ast-types/def/core.js
+++ b/node_modules/ast-types/def/core.js
@@ -132,7 +132,7 @@ module.exports = function (fork) {
         .bases("Node")
         .build("param", "guard", "body")
         // https://github.com/tc39/proposal-optional-catch-binding
-        .field("param", or(def("Pattern"), null), defaults["null"])
+        .field("param", or(def("Pattern"), null, undefined), defaults["null"])
         .field("guard", or(def("Expression"), null), defaults["null"])
         .field("body", def("BlockStatement"));
 
